server/verify_test.go

```diff
@@ -816,3 +822,136 @@ func TestGetInstanceInfoASN(t *testing.T) {
 		t.Errorf("getInstanceInfo did not return expected instance info: got %v, want %v", instanceInfo, expectedInstanceInfo)
 	}
 }
+
+func TestVerifyAttestationWithSevSnp(t *testing.T) {
```

Comment:
Could you add a few failure mode tests? For example, using bad roots, incorrect nonce, allow debug false.